linux_os/guide/system/accounts/accounts-pam/package_pam_pwquality_installed/rule.yml

```
@@ -27,10 +27,11 @@ references:
 
 ocil_clause: 'the package is not installed'
 
+ocil: |-
 {{% if 'ubuntu' not in product %}}
```

```suggestion
{{%- if 'ubuntu' not in product %}}
```

To remove the blank at the start of the OCIL block.

---

fixed by rebase.